The Economic Times.js

```diff
@@ -0,0 +1,192 @@
+{
+	"translatorID": "1a9a7ecf-01e9-4d5d-aa19-a7aa4010da83",
+	"label": "The Economic Times",
+	"creator": "Sonali Gupta",
+	"target": "http://economictimes.indiatimes.com",
```

Comment:
Let me say it better: You input the information on the first tab in Saffold and all these information will be automatically written into this part here.